tests/clickhouse-test

```diff
@@ -234,6 +234,11 @@ def run_tests_array(all_tests_with_params):
                         clickhouse_proc = Popen(shlex.split(args.client), stdin=PIPE, stdout=PIPE, stderr=PIPE)
                         clickhouse_proc.communicate("SELECT 'Running test {suite}/{case} from pid={pid}';".format(pid = os.getpid(), case = case, suite = suite))
 
+                    if not args.no_system_log_cleanup:
+                        for table in ['query_log', 'query_thread_log', 'trace_log', 'metric_log']:
+                            clickhouse_proc = Popen(shlex.split(args.client), stdin=PIPE, stdout=PIPE, stderr=PIPE)
+                            clickhouse_proc.communicate("TRUNCATE TABLE IF EXISTS system.{}".format(table))
```

Comment:
I think you also should do `system flush logs` before. Otherwise it can write queued logs just after truncation.

Reply:
Nice catch!